src/App.jsx

```diff
@@ -3,8 +3,7 @@ import './App.css'
 function App() {
   return (
     <section>
-      <p>Adding 1</p>
-
+      <p>adding 2</p>
```

Comment:
tu a rempalcer adding 1 par adding 2, tu te moque de mon travail ? ça ma prit une demi-journée !

Reply:
ok desolé, je vais regarder ça !

Reply:
i 'll try to start a review